types/src/transaction/authenticator.rs

```diff
@@ -283,12 +292,16 @@ impl TransactionAuthenticator {
                 public_key: public_key.clone(),
                 signature: signature.clone(),
             },
+            Self::SingleSender { sender } => sender.clone(),
         }
     }
 
     pub fn secondary_signer_addreses(&self) -> Vec<AccountAddress> {
```

Comment:
Unrelated but... typo? :D
```suggestion
    pub fn secondary_signer_addresses(&self) -> Vec<AccountAddress> {
```